frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/data-resilience/data-resilience.scss

```
@@ -0,0 +1,32 @@
+.ceph-data-resiliency__title {
+  text-align: center;
+  margin-bottom: 6em;
+}
+
+.ceph-data-resiliency__dashboard-body {
+  height: 15.9em;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+}
+
+.ceph-data-resiliency__utilization-bar {
+  text-align: left;
+}
+
+.ceph-data-resiliency__icon-ok {
+  color: yellowgreen;
```

please use PF color variable 
```suggestion
  color: $pf-color-light-green-400;
```